ansible/roles/uan_disk_config/README.md

```diff
@@ -136,7 +136,27 @@ This role is included in the UAN `site.yml` play.
 License
 -------
 
-Copyright 2019-2021 Hewlett Packard Enterprise Development LP
+MIT License
+
+(C) Copyright [2021] Hewlett Packard Enterprise Development LP
```

Comment:
I'm noticing that on a few of these the year is changed from 2019-2021 to 2021, but not on others. It seems to be when the year starts with 2019.

Reply:
Good catch!  Yes, several had errors in the year range.  Update being pushed shortly...

Reply:
Pushed missed date corrections.